spec/features/user_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+RSpec.describe 'User integrations', type: :feature do
+  it 'displays all users' do
+    create(:user)
+    visit users_path
+    expect(page).to have_content('Andrew Carnegie')
+  end
+
+  it 'displays user profile' do
+    user = create(:user)
+    visit user_path(user)
+    expect(page).to have_content('Andrew Carnegie')
+  end
+
+  it 'redirects to create new post page when user clicks on "New Post" link' do
+    user = create(:user)
+    create(:post, user:)
+    visit user_posts_path(user)
+    click_link 'New post'
+    expect(current_path).to eq(new_user_post_path(user))
+  end
```

Comment:
- [ ] 13 out of 13 tests failed, Please make sure that all tests are passed.
![image](https://user-images.githubusercontent.com/83803414/189369755-bb912b2c-b2d6-4c0f-9b70-18fd2b2bd077.png)

- [ ] You implemented only 13 tests, Please make sure to implement tests for all features listed [Here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=User%20index%20page,may%20have%20implemented!)